domains/pgh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "thanhnguyen2075",
+        "email": "abc.bingboong@yahoo.com.vn"
+    },
+    "record": {
+        "A": ["103.75.187.18"]
```

Comment:
I cannot search your website
I get the redirect to the `/cgi-sys/defaultwebpage.cgi` page which shows "webhosting.inet.vn refused to connect."